### PR TITLE
changed the documentation for background styling

### PR DESCRIPTION
**Pull request by erfanhamdi**
# Description
Change the documentation on block styling using js and CSS on the background color change. 
`body` to `.gradio-container`

Closes: #2270

# Checklist:

- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project
- [x] I have commented my code in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: guides/1)getting_started/2)key_features.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ Take a look at the [Docs](https://gradio.app/docs) to see all the styling option
 For additional styling ability, you can pass any CSS to your app using the `css=` kwarg:
```
Comment (abidlabs) [inline guides/1)getting_started/2)key_features.md:117]:
```suggestion
For additional styling ability, you can pass any CSS to your app using the `css=` kwarg. The base class for the Gradio app is `gradio-container`, so here's an example that changes the background color of the Gradio app:
```

File: guides/3)building_with_blocks/4)custom_CSS_and_JS.md [hunk 2]
```diff
@@ -5,14 +5,14 @@
 For additional styling ability, you can pass any CSS to your app using the `css=` kwarg:
```
Comment (abidlabs) [inline guides/3)building_with_blocks/4)custom_CSS_and_JS.md:5]:
```suggestion
For additional styling ability, you can pass any CSS to your app using the `css=` kwarg. The base class for the Gradio app is `gradio-container`, so here's an example that changes the background color of the Gradio app:
```

**Comment by abidlabs** [comment]
Thanks @erfanhamdi the code snippets look great! Would you be able to also add something in the text above the code snippets to explain the code change? I added a suggestion above

**Comment by erfanhamdi** [comment]
> Thanks @erfanhamdi the code snippets look great! Would you be able to also add something in the text above the code snippets to explain the code change? I added a suggestion above

yeah sure

**Review by aliabid94 — APPROVED** [review]
LGTM

**Pull request merged**